R/utils.R

```diff
@@ -58,7 +58,7 @@ is_installed <- function(pkg) {
 pluck_chr <- function(l, what) vapply(l, `[[`, character(1), what)
 
 is_testing <- function() {
-  identical(Sys.getenv("TESTTHAT"), "true")
+  identical(Sys.getenv("TESTTHAT"), "true") || identical(Sys.getenv("R_COVR"), "true")
```

Comment:
Why is this covr specific check necessary? `test_dir()` should set `TESTHAT` even when run in covr.

Reply:
Maybe `test_dir()` does, but `test_coverage_file()` doesn't

Reply:
Oh, that is a bug in `test_coverage_file()` then...

Reply:
Should be fixed by https://github.com/r-lib/devtools/commit/f77c14e03d48b5c79f431d649c50ec799fc6ebc7, which sets `TESTTHAT` in `test_coverage_file()`.